docs/data/upload.md

```diff
@@ -0,0 +1,113 @@
+---
+title: "Upload Data"
+linkTitle: "Upload Data"
+description: "Upload data to the Viam app from your local computer or mobile device using the data client API, Viam CLI, or Viam mobile app."
+weight: 40
+type: "docs"
+tags: ["data management", "cloud", "sync"]
+image: "/services/icons/data-capture.svg"
+imageAlt: "Upload data to the Viam app"
+images: ["/services/icons/data-capture.svg"]
+# SME: Alexa Greenberg
+---
+
+If you configured [data capture](/data/capture/) on your machine, data is automatically uploaded to the Viam cloud from the directory and at the interval you specified.
+However, if you want to upload a batch of data manually from somewhere else either on your machine, or from your personal computer or mobile device, you have several options:
+
+- Run a Python script to upload files from a folder using the data client API `file_upload_from_path` method.
+- Run a Golang script calling the data management service `sync` method.
+- Upload images from your mobile device using the Viam mobile app.
+
+## Upload data with a Python script
+
+You can use the Viam Python SDK's data client API [`file_upload_from_path`](/build/program/apis/data-client/#fileuploadfrompath) method to upload each file you'd like to upload from your computer to the Viam cloud.
```

Comment:
Nit: since we can upload multiple files as once as well (dir using your `for` loop approach), maybe:

```suggestion
You can use the Viam Python SDK's data client API [`file_upload_from_path`](/build/program/apis/data-client/#fileuploadfrompath) method to upload one or more files from your computer to the Viam cloud.
```